src/main/resources/init_db.sql

```diff
@@ -1,4 +1,43 @@
-CREATE DATABASE `internet_shop` DEFAULT CHARACTER SET utf8;
+
+
+DROP TABLE IF EXISTS `order_products`;
+
+CREATE TABLE `order_products` (
+                                  `id` bigint NOT NULL AUTO_INCREMENT,
+                                  `order_id` bigint NOT NULL,
+                                  `product_id` bigint NOT NULL,
+                                  PRIMARY KEY (`id`),
+                                  KEY `orders_order_id_fk_idx` (`order_id`),
+                                  KEY `orders_product_id_fk_idx` (`product_id`),
+                                  CONSTRAINT `orders_order_id_fk` FOREIGN KEY (`order_id`) REFERENCES `orders` (`order_id`),
+                                  CONSTRAINT `orders_product_id_fk` FOREIGN KEY (`product_id`) REFERENCES `products` (`product_id`)
+);
```

Comment:
let's create tables in such an order to not break foreign key constraints

for example: products => orders (because it uses FK to product) => orders_products
and so on

Reply:
I have used an auto export at MySql WorkBench its made this order, I was hoping it will do by the correct way. Fixed order and checked on test database, all works correctly.